netebpfext/net_ebpf_ext_bind.c

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief    This file implements the BIND program type hook on eBPF for Windows.
+ *
+ */
+
+#define INITGUID
+
+#include "net_ebpf_ext.h"
+
+// ebpf_bind_program_data.h has generated
+// headers. encode_program_info generates them from the structs
+// in ebpf_nethooks.h. This workaround exists due to the inability
+// to call RPC serialization services from kernel mode. Once we switch
+// to a different serializer, we can get rid of this workaround.
+#include "ebpf_bind_program_data.h"
```

Comment:
move up before line 11 so includes are in alphabetical order

Reply:
Fixed in #520